frontend/src/metabase-types/api/card.ts

```diff
@@ -70,6 +70,7 @@ export interface Card<Q extends DatasetQuery = DatasetQuery>
 
   creator?: CreatorInfo;
   "last-edit-info"?: LastEditInfo;
+  table_id?: TableId;
```

Comment:
WDYT about tweaking `Card` or just `createMockCard` typing so that we get an error if `type: "metric"` is used without `table_id`?